lib/recurly/bus.js

```diff
@@ -45,14 +45,12 @@ export class Bus extends Emitter {
     this.debug('message event received', messageEvent.data);
     const message = Message.createFromMessageEvent(messageEvent, this.debug.bind(this));
     if (!this.shouldHandleMessage(message)) return;
-    this.send(message, null, { exclude: [messageEvent.srcElement] });
+    this.send(message, null, { exclude: [messageEvent.srcElement, messageEvent.source] });
```

Comment:
`srcElement` is IE-only, which is the sensitive runtime this was added to protect, but it is best to generically exclude the source from the recipient list